.github/ISSUE_TEMPLATE.md

```diff
@@ -13,13 +13,13 @@ Please fill in the following sections:
 ### Describe how to reproduce or add a test
 > Describe how we can can reproduce the problem.
 
-> The *best* way is to provide a 'self containing' test, a test that can be run without dependancies. The [XQSuite - Annotation-based Test Framework for XQuery](http://exist-db.org/exist/apps/doc/xqsuite.xml) makes it very easy for you to create tests. These tests can be executed from the [eXide editor](http://exist-db.org/exist/apps/eXide/index.html) (XQuery - Run as Test)
+> The *best* way is to provide an [SSCCE (Short, Self Contained, Correct (Compilable), Example)](http://sscce.org/). One type of SSCCE could be a small test which reproduces the issue and can be run without dependencies. The [XQSuite - Annotation-based Test Framework for XQuery](http://exist-db.org/exist/apps/doc/xqsuite.xml) makes it very easy for you to create tests. These tests can be executed from the [eXide editor](http://exist-db.org/exist/apps/eXide/index.html) (XQuery - Run as Test)
 
 ### Context information
 Please always add the following information
 - eXist-db version + Git Revision hash e.g. eXist-db 3.0 /  acd0c14 
```

Comment:
it would be good to have a more easy way of filling this template: not sure how.